logger/src/lib.rs

```diff
@@ -71,6 +71,7 @@ pub fn setup_log(config: &Config) -> Result<Arc<RotatingLogger>, String> {
 	builder.filter(Some("ws"), LevelFilter::Warn);
 	builder.filter(Some("hyper"), LevelFilter::Warn);
 	builder.filter(Some("rustls"), LevelFilter::Error);
+	builder.filter(Some("rpc"), LevelFilter::Trace);
```

Comment:
Used for testing because the log can't be configured from the `C lib`. Should be removed before merging

Reply:
Will keep it, if people want to test it helps to see what happens :P

Reply:
@niklasad1 But this will propagate to release build, wouldn't it? I think we still need to remove this before merging, right?

Reply:
Yeah, it will be removed :smile: